corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -434,6 +437,7 @@ hqDefine('app_manager/js/releases/releases', function () {
         };
         self.actuallyMakeBuild = function () {
             self.buildState('pending');
+            self.errorMessage(self.genericErrorMessage);
```

Comment:
Nitpick: it's a little weird to set the error message here, when there isn't necessarily an error. I'd have expected this as an `else` in the error handler below.

Reply:
It's to reset things before starting a request so that in case of two simultaneous requests the the error message from last request is reset to default.
Since we are using that message as the default message i set it here just like build state instead of else block below.

Reply:
Plus the error message wont be visible until there is an error which is not 403, hence the setting to default even initialising object.

Reply:
Yeah, I realize it isn't visible, it's just odd to read. Fine to leave as-is.